book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -8,24 +8,324 @@ This can involve hosting different events that are attended by people with lived
+In particular, actively involving a community representative or person with lived experience in the event planning process will help to ensure both a diverse perspective, and that the accessibility requirements unique to your attendees are identified. 
+
+```{note}
+If you are able to involve a member of the community, please consider how you will recognize or reimburse the time and contributions made.
+This may involve an honorarium for attending planning meetings and for time spent reviewing policies and other documents.
+You could also consider hosting some focus group sessions at regular stages to get feedback on your plans.
+Where possible, the expertise and labor of people with lived experience should be recognized and renumerated. 
+```
+
+
+To identify the event's purpose and outcomes, carefully consider your target audience and the reasons behind their inclusion. 
+It is important at this stage to define and document the purpose of public involvement in the event alongside a representative or member of the public with lived experience (cite: NIHR public involvement guidance). 
+
+It is important to clearly define the event's goals, understand the intended audience and potential attendees, and to emphasize the importance of including people with lived experiences throughout the event. 
+
+During the initial planning, organiziers can prioritize inclusion by deciding how accessibility requirements are going to be met at a level suitable for all attendees. 
+This suitable level should be discussed and defined by people with lived experience and the specific accessibility need. 
+This includes considering physical and cognitive accessibility, communication needs, and any other accommodations necessary. 
+
+Finally, when developing the event agenda, make sure it is inclusive of all attendees. Consider various perspectives, backgrounds, and needs, to create an agenda that fosters an inclusive and engaging experience for everyone involved.
+
+(cl-inclusive-events-power-dynamics)=
+### Addressing Power Dynamics
+Acknowledging and addressing the power and privilege differences between members of the public and researchers is an important part of creating an inclusive environment where people with lived experience are able to engage equitably at an event. 
+
+It can be helpful to acknowledge that people with lived experience are "experts by experience" to help establish equal footing with professionals and researchers who are "experts by profession". 
+
+Your event may be the first event of this type attended by people with lived experience and members of the public, and they may not have pursued higher education compared with attendees who may be researchers by profession. 
+This can create a significant power imbalance due to comfort and experience at the event. Networking and social components of the event may feel less welcoming to people with lived experience if they do not know anyone before attending. 
```

Comment:
```suggestion
This can create a significant power imbalance due to comfort and experience at the event. 
Networking and social components of the event may feel less welcoming to people with lived experience if they do not know anyone before attending. 
```